ui/src/app/richskill/service/rich-skill.service.ts

```diff
@@ -149,6 +149,30 @@ export class RichSkillService extends AbstractService {
       }))
   }
 
+  libraryExport(
+    publishStatus: Set<PublishStatus>,
```

Comment:
Why are we providing search and paging params to a request to dump the entire library?

Reply:
I asked this too, but Jose mentioned he created the API route with those params, so i'm just passing what he requested.